testcharms/charm-hub/charms/juju-qa-test/.jujuignore

```diff
@@ -4,3 +4,10 @@
 charmcraft.yaml
 build
 foo-file.txt
+edge/
+edge/version
```

Comment:
I don't think you need to specify these individually if the directory itself is ignored, do you?